rocAL/rocAL_pybind/examples/prefetch_queue_depth/prefetch_queue_depth.py

```diff
@@ -0,0 +1,52 @@
+from amd.rocal.plugin.pytorch import ROCAL_iterator
```

Comment:
Why do we need to use the pytorch iterator? can we not use the generic one? This will again complain if pytorch docker is not used

Reply:
Done